crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,6 +1156,19 @@ impl World {
         result
     }
 
+    /// "Sends" an `event`.
```

Comment:
Why is “Sends” in quotes? `event_send_default` doesn't do that.

Reply:
I copied this from the doc comments on the event method. Agree this isn't useful.